transform/transformers/mbs_transformer.py

```diff
@@ -185,12 +185,12 @@ def get_identifiers(self, batch_nr=0, seq_nr=0):
             "period": self.response.get("collection", {}).get("period"),
         }
 
-        if any(i is None for i in ids):
-            logger.error("Missing an id from {0}".format(ids))
-            return None
+        for key, value in ids.items():
+            if value is None:
+                logger.error("Missing value for: {0}".format(key))
```

Comment:
Probably want to check for empty values as well with: `if not value:`.

Reply:
We can have empty values with `seq_nr` and maybe one or two others, so this is only checking for `None` values